cmd/utils/flags.go

```diff
@@ -1681,6 +1681,7 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 	switch {
 	// CHANGE(taiko): when `--taiko` flag is set, use the Taiko genesis.
 	case ctx.IsSet(TaikoFlag.Name):
+		core.InternalShastaTime = ctx.Uint64(TaikoInternalShastaTimeFlag.Name)
```

Comment:
We still need to set https://github.com/taikoxyz/taiko-geth/blob/taiko/core/taiko_genesis.go#L25 to this `core.InternalShastaTime`?

Reply:
that's the variable in core package